src/main/java/com/czertainly/core/service/impl/LocationServiceImpl.java

```diff
@@ -774,7 +775,7 @@ private ClientCertificateDataResponseDto issueCertificateForLocation(Location lo
         ClientCertificateSignRequestDto clientCertificateSignRequestDto = new ClientCertificateSignRequestDto();
         clientCertificateSignRequestDto.setAttributes(issueAttributes);
         // TODO: support for different types of certificate
-        clientCertificateSignRequestDto.setPkcs10(csr);
+        clientCertificateSignRequestDto.setRequest(csr);
```

Comment:
I do not think the default `format` will be assigned here because we are not using builder as with other DTOs.